db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+# Race model
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+# Skill model
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.IntegerField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is an `IntegerField`, which is correct for storing numerical values. However, `IntegerField` does not support the `max_length` attribute. You should remove `max_length=255` from this field.